src/cmd/mapper/mapper-upload.go

```diff
@@ -21,19 +21,23 @@ var UploadCmd = &cobra.Command{
 			}
 			ctxTimeout, cancel := context.WithTimeout(context.Background(), config.DefaultTimeout)
 			defer cancel()
-			intentsClient := intents.NewClientFromToken(viper.GetString(config.OtterizeAPIAddressKey), config.GetAPIToken(ctxTimeout))
+			intentsClient := graphql.NewClientFromToken(viper.GetString(config.OtterizeAPIAddressKey), config.GetAPIToken(ctxTimeout))
```

Comment:
Can you delete this command? There's a task for it: https://www.notion.so/otterize/Remove-active-intents-upload-from-the-mapper-f0da4ba2f74d4db0ab78fb3edfd82207